xwiki-platform-core/xwiki-platform-ratings/xwiki-platform-ratings-ui/src/main/resources/XWiki/Ratings.xml

```diff
@@ -446,20 +446,22 @@ position: relative;
     </class>
     <property>
       <content>{{velocity}}
-#set($canView = $services.security.authorization.hasAccess("view", "XWiki.RatingsMacros"))
+#set($canView = $services.security.authorization.hasAccess('view', 'XWiki.RatingsMacros'))
 #if($canView)
   {{include reference='XWiki.RatingsMacros'/}}
-  #set($configurationClass = 'XWiki.RatingsConfigClass')
+#end
+{{/velocity}}
```

Comment:
Follow up question: Why can a user see `XWiki.Ratings` but not `XWiki.RatingsMacros`? It looks like inconsistent rights to me.

New question... Why are there two files, `rating_macros.vm` and `XWiki.RatingsMacros`? Can't the wiki macros be placed in the template file?  Or the other way around?